src/ui/deleteAccount/ViewDeleteAccount.qml

```diff
@@ -11,13 +11,41 @@ import components.forms 0.1
 import components.inAppAuth 0.1
 
 Item {
+    property bool _menuVisible: false
+    property string prevState: ""
+
     id: viewDeleteAccount
 
     Loader {
         id: loader
 
-        asynchronous: true
         anchors.fill: parent
+
+        asynchronous: true
+        onStatusChanged: {
+            if (loader.status !== Loader.Ready) {
+                return;
+            }
+            
+            if (
+                prevState === ""
+                && viewDeleteAccount.state === "StateInitializing"
+            ) {
+                // `StateInitializing` is set for the first time
```

Comment:
I don't understand this status check. This function. Why do we need it? Let's talk on slack/zoom

Reply:
As discussed `isDeleteAccountAuth` is now set by default for all views.